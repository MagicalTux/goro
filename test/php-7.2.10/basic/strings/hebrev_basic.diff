     *** Testing hebrev() : basic functionality ***
002- string(109) ".The hebrev function converts logical Hebrew text to visual text
003- .The function tries to avoid breaking words
004- "
005- string(109) "to visual text
006- Hebrew text
007- logical
008- converts
009- hebrev function
010- .The
011- breaking words
012- tries to avoid
013- .The function
014- "
015- ===DONE===
002+ 
003+ Fatal error: Uncaught Error: Call to undefined function hebrev in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/hebrev_basic.php:12
004+ Stack trace:
005+ #0 {main}
006+   thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/hebrev_basic.php on line 12
