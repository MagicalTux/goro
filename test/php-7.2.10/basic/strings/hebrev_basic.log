
---- EXPECTED OUTPUT
*** Testing hebrev() : basic functionality ***
string(109) ".The hebrev function converts logical Hebrew text to visual text
.The function tries to avoid breaking words
"
string(109) "to visual text
Hebrew text
logical
converts
hebrev function
.The
breaking words
tries to avoid
.The function
"
===DONE===
---- ACTUAL OUTPUT
*** Testing hebrev() : basic functionality ***

Fatal error: Uncaught Error: Call to undefined function hebrev in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/hebrev_basic.php:12
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/hebrev_basic.php on line 12
---- FAILED
