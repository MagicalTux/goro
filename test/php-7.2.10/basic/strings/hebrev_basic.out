*** Testing hebrev() : basic functionality ***

Fatal error: Uncaught Error: Call to undefined function hebrev in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/hebrev_basic.php:12
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/hebrev_basic.php on line 12